docs/deployment/k8s/index.mdx

```diff
@@ -10,26 +10,30 @@ For high-level discussion of our approach to Kubernetes and compatibility with c
 
 On **this** page, you can find documentation for operators wishing to deploy wasmCloud with Kubernetes, including details on prerequisites such as NATS and how to deploy the operator.
 
+:::info[kubectl support]
+When running wasmCloud on Kubernetes, you can use `kubectl` to deploy [wasmCloud manifests](/docs/ecosystem/wadm/model), check wasmCloud logs, and more. The example below demonstrates standard usage of `kubectl` with wasmCloud.

```

Comment:
I'm wondering if we've settled on the terminology here, I know we've called them "wadm applications" and "wasmcloud applications" interchangeably (or maybe that's just me?) in the past, but I don't know if we've got a preferred term to use.

Reply:
by here I'm referencing `wasmCloud manifests` specifically

Reply:
My 2 cents is that "wasmCloud application manifests" is preferable. If you're new to the platform (running through the quickstart, say) understanding exactly what wadm is, what it stands for, etc probably shouldn't be a conceptual prerequisite for using a declarative manifest to deploy a thing on wasmCloud.